cypress/e2e/participant/default-tests/playCVA.cy.js

```diff
@@ -4,13 +4,26 @@ import { isCurrentVersion } from '../../../support/utils';
 const app = '@bdelab/roar-multichoice';
 
 describe('ROAR - Written Vocabulary Playthrough', () => {
-  it('Plays CVA', () => {
-    cy.wrap(isCurrentVersion(app)).then((isCurrentVersion) => {
+  it('Plays CVA with username/password combo', () => {
+    cy.wrap(!isCurrentVersion(app)).then((isCurrentVersion) => {
```

Comment:
```suggestion
    cy.wrap(isCurrentVersion(app)).then((isCurrentVersion) => {
```
Is "!" just to mock-run the tests?

Reply:
That would be correct.